packages/nodes-base/nodes/Form/utils.ts

```diff
@@ -140,8 +140,11 @@ const checkResponseModeConfiguration = (context: IWebhookFunctions) => {
 	}
 };
 
-export async function formWebhook(context: IWebhookFunctions) {
-	const nodeVersion = context.getNode().typeVersion;
+export async function formWebhook(
+	context: IWebhookFunctions,
+	authProperty = FORM_TRIGGER_AUTHENTICATION_PROPERTY,
```

Comment:
This is why I brought up [this](https://n8nio.slack.com/archives/C036PA5D42D/p1721662802954819) conversation. If we switch to using class  inheritance (instead of the custom style we use), we wouldn't need to pass data around like this, and we could use `this.authPropertyName`, like we do in Webhook and Wait nodes.